app/routes/apply/new-eligibility/benefits.js

```diff
@@ -14,21 +24,34 @@ module.exports = function (router) {
   router.post('/apply/:claimType/new-eligibility/:dob/:relationship', function (req, res) {
```

Comment:
post url shouldn't have the session values, this needs to be changed in the view too